ironic-config/ironic.conf.j2

```diff
@@ -30,6 +30,15 @@ host = localhost
 host = {{ env.IRONIC_URL_HOST }}
 {% endif %}
 
+# If a path to a certificate is defined, use that first for webserver
+{% if env.WEBSERVER_CACERT_FILE %}
+webserver_verify_ca =  {{ env.WEBSERVER_CACERT_FILE }}
+{% elif env.IRONIC_INSECURE == "true" %}
+webserver_verify_ca = false
+{% else %}
+webserver_verify_ca = true
```

Comment:
nit: no need for this branch, true is the default

Reply:
Done